src/sentry/grouping/strategies/legacy.py

```diff
@@ -88,7 +95,7 @@ def is_recursion_legacy(frame1, frame2):
     return True
 
 
-def remove_module_outliers_legacy(module, platform):
+def remove_module_outliers_legacy(module: str, platform: str) -> Tuple[str, Optional[str]]:
     """Remove things that augment the module but really should not."""
```

Comment:
Any benefit in enumerating the types of action performed ?
There are a few remove functions like this.

I'm thinking about something like


```
from typing import Litereal, Optional

RemoveOperation = Literal["removed reflection marker", "removed codegen marker"]

def remove_module_outliners(...) -> Tuple[str, Optional[RemoveOperation]]
....
```

I also prefer to use a named tuple (which I define just above the function) to make it clear what the tuple
contains something like:

```
RemoveOpRetVal = namedtuple("ReomveOpRetVal", "name, change_op")
```

I would probably not use both named tuple and RemoveOperation together, only one would make it reasonably clear what is returned.

Reply:
This would be a runtime change, I want to keep the type annotations separate from changing runtime types